x-pack/plugins/security_solution/public/timelines/components/timeline/expandable_event/index.tsx

```diff
@@ -52,6 +53,15 @@ const StyledEuiFlexGroup = styled(EuiFlexGroup)`
   flex: 0;
 `;
 
+const StyledFlexGroup = styled(EuiFlexGroup)`
+  height: 100%;
+`;
+const TabListHeight = 48;
+const StyledEuiFlexItem = styled(EuiFlexItem)<{ $tabListHeight?: number; $messageHeight?: number }>`
```

Comment:
can we try to achieve the same just with the css? using ref can have impact on the performance